config.toml.example

```diff
@@ -392,7 +398,7 @@
 # desired in distributions, for example.
 #rpath = true
 
-# Emits extra output from tests so test failures are debuggable just from logfiles.
+# Emits extra output from the tests to debug issues in the test harness itself.
```

Comment:
IMO instead of "extra output" we should say it prints out the test names, as they're being tested.